ChiModules/LinearBoltzmannSolver/Acceleration/diffusion_mip_00_constrdestr.cc

```diff
@@ -0,0 +1,48 @@
+#include <utility>
+
+#include "diffusion_mip.h"
+
+#include "ChiMath/SpatialDiscretization/spatial_discretization.h"
+
+//###################################################################
+/**Default constructor.*/
+lbs::acceleration::DiffusionMIPSolver::
+  DiffusionMIPSolver(std::string text_name,
+                     const chi_mesh::MeshContinuum &grid,
+                     const chi_math::SpatialDiscretization& sdm,
+                     const chi_math::UnknownManager& uk_man,
+                     std::vector<BoundaryCondition>  bcs,
+                     MapMatID2XS map_mat_id_2_xs,
+                     const std::vector<UnitCellMatrices>& unit_cell_matrices,
+                     const bool verbose/*=false*/) :
+  m_text_name(std::move(text_name)),
```

Comment:
for my education, why do you choose that all names from the constructor have 'm_' ?

Reply:
I probably needed to do this from the start. It shows that a variable is a member. It is good for someone with a fairly "dumb" IDE. I've never done it because all the IDEs I've used highlighted members as a different color.

In the end we will find people who likes this and people who don't so I believe we should just allow either way.